R/viz.R

```diff
@@ -4603,24 +4736,22 @@ fancy_alignment_plot = function(these_samples,
 #' @export
 #'
 #' @examples
-#' #plot QC metrics (Average base Quality for all BL samples), compared to the same metric for all FL cases (pathology).
-#' qc_avg_basequality = fancy_qc_plot(filter_pathology = "FL",
-#'                                    filter_cohort = "DLBCL_LSARP_Trios",
-#'                                    filter_comp_cohort = "BL_Pediatric",
-#'                                    filter_comp_pathology = "BL",
-#'                                    sort_by = "AverageBaseQuality",
-#'                                    plot_data = "AverageBaseQuality",
-#'                                    fill_by = "pathology",
-#'                                    plot_title = "Average Base Quality",
-#'                                    y_axis_lab = "Base Quality",
-#'                                    seq_type = "genome")
+#' fancy_qc_plot_new(filter_pathology = "FL",
```

Comment:
is the name of the function in this example supposed to stay as is (so no _new) ?

Reply:
That one slipped through the cracks. Good catch!